src/libssh2_priv.h

```diff
@@ -50,6 +50,11 @@
 #undef WIN32_LEAN_AND_MEAN
 #endif
 
+/* Windows UWP/WinRT platform. */
+#if defined(WINAPI_PARTITION_APP) && WINAPI_PARTITION_APP
+#define WINAPP
```

Comment:
If this is a custom define, we should probably prefix it with `LIBSSH2_`

Reply:
@mback2k, sorry noticed your reply only now when receive a e-mail notification about PR closure. It seems corrections were incorporated anyway here https://github.com/libssh2/libssh2/blob/730c606b64f86b5899183655e8f07578a54b8f9e/src/agent_win.c#L58